proposals/devcontainer-features.md

```diff
@@ -151,11 +151,11 @@ This property is declared by the user in their `devcontainer.json` file.
 
 Any feature IDs listed in this array will be installed before all other features, in the provided order. Any omitted features will be installed in an order selected by the implementing tool, or ordered via the `installsAfter` property _after_  any features listed in the `overrideFeatureInstallOrder` array, if applicable. 
 
-All feature `id` provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`.
+All feature `id` provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`. Also, all the feature IDs which follows the OCI registry format, the `id` must be referenced without the semantic version in `overrideFeatureInstallOrder`.
```

Comment:
```suggestion
All unversioned feature `id`s provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`. For instance, all the features which follows the OCI registry format would include everything except for the label that contains the version (`<oci-registry>/<namespace>/<feature>` without the `:<semantic-version>`.
```

It might be good to also include an example:

```jsonc
"features": {
  "ghcr.io/devcontainers/features/node:1": {},
  "ghcr.io/devcontainers/features/go:1": {}
},
"overrideFeatureInstallOrder": [
  "ghcr.io/devcontainers/features/go" // Force the 'go' feature to be installed first
]
```

Reply:
Updated and added ex. Thanks!